### DISCUSSION TITLE
ExtremelyFastDecisionTreeClassifier: How can I input multiple samples in a single learning session?

### DISCUSSION THREAD

**Discussion by 616156** [online-ml/river#1312] [Q&A] [answered]
I'm implementing a project where input samples are divided into two categories. First, one portion of the data is used for training, and then, after completion, the other portion of the data is used for training. I need to utilize the incremental learning feature of the ExtremelyFastDecisionTreeClassifier, but the efficiency is too low when inputting only one sample at a time. 

Is there any way to enable the ExtremelyFastDecisionTreeClassifier to input multiple pieces of data in a single learning session?

**Comment by smastelini** [comment, marked as answer]
Hi! Unfortunately the tree module doesn't support mini batching operations at the moment.
